bioptim/examples/stochastic_optimal_control/arm_reaching_muscle_driven.py

```diff
@@ -608,9 +617,11 @@ def prepare_socp(
         dynamics,
         n_shooting,
         final_time,
+        t_init=t_init,
```

Comment:
Relevence of having initial guess and bound for time as it is driven by ocp.node_time ?
Was implemented when time was variable container